src/js/tourFinder.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * The method that get the user Geolocation, if the user open the Tour Finder page.
+ */
+
+$(document).ready(function() {
+    if (window.location.pathname == '/tour-finder.html') {
```

Comment:
you should try to load this file only on the tour finder page and this way you no longer have to check on which page you are. for each page/ component you should have a separate js file (see example in the extranet implementation)